src/docfx/Errors.cs

```diff
@@ -52,7 +52,10 @@ public static Error YamlHeaderNotObject(object filePath, bool isArray)
             => new Error(ErrorLevel.Warning, "yaml-header-not-object", $"Expect yaml header to be an object, but got {(isArray ? "an array" : "a scalar")}", filePath.ToString());
 
         public static Error YamlSyntaxError(Exception ex)
-            => new Error(ErrorLevel.Error, "yaml-syntax-error", ex.Message);
+            => new Error(ErrorLevel.Error, "yaml-syntax-error", $"{ex.Message}. {ex.InnerException?.Message}");
```

Comment:
>Message [](start = 70, length = 7)

can we extract line column info from YamlSyntaxException and put them to Error ?

Reply:
Yes, we can. But it wouldn't make any difference for this one, would simplify refinement for the duplicate key

---
In reply to: [200893825](https://github.com/dotnet/docfx/pull/3009#discussion_r200893825) [](ancestors = 200893825)

Reply:
so the *original* error message with syntax error is clear regarding line/column info?

---
In reply to: [200897131](https://github.com/dotnet/docfx/pull/3009#discussion_r200897131) [](ancestors = 200897131,200893825)

Reply:
We only redefined the error message of duplicate key for now, keep the rest as they are. If we want to redefine more, we can do it accordingly.

---
In reply to: [200916391](https://github.com/dotnet/docfx/pull/3009#discussion_r200916391) [](ancestors = 200916391,200897131,200893825)

Reply:
misunderstood above, yes, the inner error message contains clear line/column info

---
In reply to: [200916391](https://github.com/dotnet/docfx/pull/3009#discussion_r200916391) [](ancestors = 200916391,200897131,200893825)